recipes/dicto/meta.yaml

```diff
@@ -0,0 +1,36 @@
+{% set name = "dicto" %}
+{% set version = "0.0.20" %}
+
+package:
+  name: "{{ name|lower }}"
+  version: "{{ version }}"
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: 82557817b4c14d74497d98a99916249685c1a770fc35ff556ebebd2d2b5fad3a
+
+build:
+  number: 0
+  script: "{{ PYTHON }} -m pip install . --no-deps --ignore-installed -vv "
+
+requirements:
+  host:
+    - pip
+    - python
+  run:
+    - python
+
+test:
+  imports:
+    - dicto
+
+about:
+  home: https://github.com/cgarciae/dicto
+  license: MIT
+  license_family: MIT
+  license_file: LICENSE 
```

Comment:
Sorry, it seems the file is not part of the pypi package. See https://github.com/conda-forge/staged-recipes/pull/7432